navbar-list.html

```diff
@@ -188,20 +188,19 @@ <h2 class="pf-vertical-sub-nav__title">Home and Garden</h2>
     <div
       id="technology-warning"
       aria-hidden="true"
-      aria-labelledby="technology-warning-title"
-      aria-live="polite"
-      role="alertdialog"
       class="pf-c-toast pf-is-warning">
-      <div class="pf-c-toast__icon">
-        <i class="fas fa-home"></i>
-        <span id="technology-warning-title" class="sr-only">Technology Warning:</span>
-      </div>
-      <div id="technology-warning-message" class="pf-c-toast__message">
-        Technology is scary, are you sure you want to proceed?
+      <div role="alert" aria-live="assertive">
+        <div class="pf-c-toast__icon">
+          <i class="fas fa-home"></i>
+          <span id="technology-warning-title" class="sr-only">Warning message:</span>
+        </div>
+        <div id="technology-warning-message" class="pf-c-toast__message">
+          Technology is scary, and may include topics that are confusing.
+        </div>
       </div>
       <div class="pf-c-toast__action">
```

Comment:
Considering the above comment about alertdialog, one problem that comes with moving the role attribute down to only part of the notification is that `.pf-c-toast__action` and its children are no longer considered part of the alertdialog, so after VoiceOver reads the actual alert message, it doesn't then automatically then read the first form control as (I _believe_) it should. See the following two comments in the below article;

> When the dialog is correctly labeled and focus is moved to a control inside the dialog, screen readers should announce the dialog's accessible role, name and optionally description before announcing the focused element.

Directly followed by

> Note: Opinions may differ on how assistive technology should handle this technique. The information provided above is one of those opinions and therefore not normative.

https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_dialog_role#Focus_management

I think it's safe to say we're approaching the brink of the capabilities here and there is no "right" way to do it. Would be great to capture this as an option to consider with the rest of the group - and just make the best guess for next GMS session. Either is probably just fine. In general though, my gut tells me we'll want to raise the scope of the alert role just enough as necessary to contain any form controls or other actions elements that may exist. In this way, we can recycle a single container instance from the JS side.

Reply:
I added another variation that uses `role="dialog"` for the parent `<div>`, but `role="alert"` is still only applied to just the message. My assumption about toast notifications is that they're not expected to take the user's focus from where they are, which is why I use `role="alert"` for just the message. And we can test if for the parent wrapper whether `<article>` or `<div role="dialog">` is the best choice.

Reply:
Ahh ok I see now, thanks for explaining. The new variation you added with `role="dialog"` on the container, unlike the other toast notifications, reads the contents of the notification upon focusing the "Stop it" link. First read is the screen reader only text, then VO continues with the notification body - which on the surface seems like what we may be going for?

It's definitely the `role="dialog"` that gives this effect.

Was there any intention or expectation of what happens upon focusing the individual notifications?

Reply:
I don't have any opinions yet. For now, I just wanted to present some variations, and capture feedback from the test participant about their experience with each variation.